client/my-sites/domains/components/domain-warnings/pending-gsuite-tos-notice.jsx

```diff
@@ -126,7 +126,7 @@ class PendingGSuiteTosNotice extends React.PureComponent {
 				siteSlug={ this.props.siteSlug }
 				user={ users[ 0 ] }
 				isCompact={ false }
-				cta={ translate( 'Get started' ) }
+				cta={ translate( 'Finish setup' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **27** times:
`translate( 'Finish Setup' )` *ES Score: 12*